src/Mod/Draft/draftobjects/patharray.py

```diff
@@ -119,7 +119,13 @@ def setProperties(self,obj):
             obj.addProperty("App::PropertyVector","VerticalVector","Alignment", _tip)
             obj.VerticalVector = App.Vector(0,0,1)
 
-        if hasattr(obj, "use_link"):
+        if self.use_link:
+            _tip = _tr("Show array element as children object")
+            obj.addProperty("App::PropertyBool","ExpandArray", "Parameters", _tip)
```

Comment:
Don't add this. You are adding the property twice again.

Reply:
I obviously don't understand what's going on so I'll just close the PR and let you submit it properly.